textile/features.textile

```diff
@@ -2493,6 +2612,22 @@ class ObjectData // OD*, internal
   encoding: String? // OD2b
   value: (String | Number | Boolean | Binary)? // OD2c
 
+class Annotation // TAN*
+  +fromEncoded(JsonObject, ChannelOptions?) -> Annotation // TAN3
+  +fromEncodedArray(JsonArray, ChannelOptions?) -> [Annotation] // TAN3
+  id: String // TAN2a
+  action: AnnotationAction // TAN2b
+  clientId: String? // TAN2c
+  name: String? // TAN2d
+  count: Int? // TAN2e
+  data: Data? // TAN2f
+  encoding: String? // TAN2g
+  timestamp: Time // TAN2h
+  serial: String // TAN2i
+  messageSerial: String // TAN2j
```

Comment:
```suggestion
  messageSerial: String // TAN2i
  seedMessageSerial: String // TAN2j
```

Reply:
will be updated in SDK models, closing as per https://github.com/ably/specification/pull/292#discussion_r2135598426